### PR TITLE
Make Glimmer work with @Turbo87's typescript branch

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

File: ember-cli-build.js [hunk 1]
```diff
@@ -85,10 +85,12 @@ module.exports = function(_options) {
...
-  var tokenizerPath = path.join(require.resolve('simple-html-tokenizer'), '..', '..', 'src');
-  // TODO: WAT, why does { } change the output so much....
-  var tokenizerTree = find(tokenizerPath, { });
-  var HTMLTokenizer = mv(tokenizerTree, "simple-html-tokenizer");
+  var tokenizerPath = path.join(require.resolve('simple-html-tokenizer'), '..', 'es6');
+  var HTMLTokenizer = find(tokenizerPath, {
+    include: ['**/*.js'],
+    exclude: ['**/*.d.ts']
+  });
+  HTMLTokenizer = mv(htmlTokenizerTree, 'simple-html-tokenizer');
```
Comment (Turbo87) [inline ember-cli-build.js]: @locks why didn't you keep the old names and where is `htmlTokenizerTree` coming from?

File: package.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
     "handlebars": "^3.0.2",
     "qunit": "^0.9.1",
     "simple-dom": "^0.3.0",
-    "simple-html-tokenizer": "^0.3.0"
+    "simple-html-tokenizer": "git://github.com/turbo87/simple-html-tokenizer.git#typescript"
```
Comment (Turbo87) [inline package.json]: this is obviously temporary, right? 😉

Comment (locks) [inline package.json]: yup :P

File: ember-cli-build.js (hunk 1 above)
Comment (locks) [inline ember-cli-build.js]: @Turbo87 I thought I did 🤔

File: package.json (hunk 2 above)
Comment (krisselden) [inline package.json]: even if temporary, git urls don't publish the npm module, so if it relies on republish this will give you problems.

Comment (locks) [inline package.json]: I only came to this conclusion after a bit of nudging from robert ;X It is working locally with everything linked, up to Ember.

Comment (krisselden) [inline package.json]: locally you've probably run prepublish but in travis this won't happen.

Comment (rwjblue) [inline package.json]: Please update to simple-html-tokenizer@0.4.0 now that it is published

**Comment by rwjblue** [comment]
Oh, and sqaush...

**Pull request merged**